Framework/PythonInterface/mantid/kernel/src/Converters/CloneToNumpy.cpp

```diff
@@ -58,9 +58,10 @@ PyObject *clone1D(const std::vector<Types::Core::DateAndTime> &cvector) {
   for (Py_intptr_t i = 0; i < dims[0]; ++i) {
     void *itemPtr = PyArray_GETPTR1(nparray, i);
     npy_datetime abstime = Converters::to_npy_datetime(cvector[i]);
-    PyArray_SETITEM(
-        nparray, reinterpret_cast<char *>(itemPtr),
-        PyArray_Scalar(reinterpret_cast<char *>(&abstime), descr, nullptr));
+    auto scalar =
+        PyArray_Scalar(reinterpret_cast<char *>(&abstime), descr, nullptr);
+    PyArray_SETITEM(nparray, reinterpret_cast<char *>(itemPtr), scalar);
+    Py_DecRef(scalar);
```

Comment:
Can we use the macro `Py_DECREF()` instead. The Python C/API docs suggest using this instead of using `Py_DecRef` directly.